### DISCUSSION TITLE
signedTypedData_v4

### DISCUSSION THREAD

**Discussion by trevormil** [wevm/wagmi#2233] [Ideas]
Currently, there is only support for signedTypedData.

It would be great if there was also support for signedTypedData_v4 which many dApps use like mine

**Comment by tmm** [comment]
`useSignTypedData`/`signTypedData` does use `signedTypedData_v4` via [ethers](https://github.com/ethers-io/ethers.js/blob/v5.7/packages/providers/src.ts/json-rpc-provider.ts#L345) and soon via [viem](https://viem.sh/docs/actions/wallet/signTypedData.html).
